hastory-server-data/src/Hastory/Server/Data.hs

```diff
@@ -25,5 +25,7 @@ ServerEntry
     dateTime UTCTime
     user Text
     hostName Text
+    contentHash Text
```

Comment:
I think we want a `ByteString` here, not a `Text`.

Reply:
Yes. I changed this to `Digest SHA256`, and made orphaned instances of `PersistField` and `PersistFieldSql` that use the underlying instances for `ByteString`.